pom.xml

```diff
@@ -200,7 +200,7 @@
 
         <version.org.jboss.spec.javax.jms.jboss-jms-api_2.0_spec>1.0.1.Final</version.org.jboss.spec.javax.jms.jboss-jms-api_2.0_spec>
         <version.org.apache.activemq.broker>5.13.3</version.org.apache.activemq.broker>
-        <version.org.jgroups>3.6.13.Final</version.org.jgroups>
+        <version.org.jgroups>4.0.11.Final</version.org.jgroups>
```

Comment:
For the record, [WildFly 12.0.0.Final uses `4.0.10.Final`](https://github.com/wildfly/wildfly/blob/12.0.0.Final/pom.xml#L236). I guess you have reasons not to be aligned exactly this time.